corehq/apps/reports/standard/monitoring.py

```diff
@@ -1629,11 +1630,11 @@ def _rows_by_group(self, report_data):
                     sum(
                         [int(report_data.avg_submissions_by_user.get(user["user_id"], 0))
                         for user in users]
-                    ) / self.num_avg_intervals
+                    ) // self.num_avg_intervals
                 ),
                 # Active users
                 util.numcell("%s / %s" % (active_users, total_users),
-                             value=int((float(active_users) / total_users) * 10000) if total_users else -1,
+                             value=int((float(active_users) // total_users) * 10000) if total_users else -1,
```

Comment:
I think this one should not be using integer division (it wasn't before).

Reply:
Good catch: 4c0cf5e